tests/src/Unit/Service/Context/PageContextTest.php

```diff
@@ -348,14 +436,21 @@ public function testPopulateHtmlHeadWithNodeAndFields() {
       ->willReturn($this->getNode());
     $this->populateHtmlHeadWithNodeAndFieldsSetUpFields();
 
-    $page_context = new PageContext($this->configFactory, $this->entityTypeManager, $this->requestStack, $this->routeMatch, $this->titleResolver);
+    // Language mock
+    $this->languageManager
+      ->expects($this->any())
+      ->method('getCurrentLanguage')
+      ->willReturn($this->languageObject);
+
+    $page_context = new PageContext($this->configFactory, $this->entityTypeManager, $this->requestStack, $this->routeMatch, $this->titleResolver, $this->languageManager);
     $page = [];
     $page_context->populate($page);
 
     $expected_head = $this->toRenderArray([
       'content_title' => 'My Title',
       'content_type' => 'article',
       'page_type' => 'node page',
+      'context_language' => 'en',
```

Comment:
I would not set or assert on `'en'` because it is Drupal default. I would go for `'fr'` or another.

Reply:
Changed to fr :)